sample/src/main/java/com/davidecirillo/multichoicesample/sampleToolbar/MySampleToolbarAdapter.java

```diff
@@ -89,4 +89,11 @@ public void onClick(View v) {
     public int getItemCount() {
         return mList.size();
     }
+
+    public void swap(ArrayList<String> data) {
+        mList.clear();
+        mList.addAll(data);
+
+        refreshDataSet();
+    }
 }
```

Comment:
In theory what should happen is the reference to the list should be kept and update the item and then call notifyDataSetChange() and it'll handle it. That is why i gave that similar name to my method.